Sources/AlgoliaSearch-Client/FilterBuilder/Proxies/OrGroupProxy.swift

```diff
@@ -0,0 +1,113 @@
+        filterBuilder.addAll(filters: filters, to: group)
+    }
+    
+    /// Tests whether group contains a filter
+    /// - parameter filter: sought filter
+    public func contains(_ filter: T) -> Bool {
+        return filterBuilder.contains(filter, in: group)
+    }
+    
+    /// Removes filter from current group and adds it to destination conjunctive group
+    /// - parameter filter: filter to move
+    /// - parameter destination: target group
+    /// - returns: true if movement succeeded, otherwise returns false
+    public func move(_ filter: T, to destination: AndFilterGroup) -> Bool {
+        return filterBuilder.move(filter: filter, from: group, to: AnyFilterGroup(destination))
+    }
+    
+    /// Removes filter from current group and adds it to destination disjunctive group
+    /// - parameter filter: filter to move
+    /// - parameter destination: target group
+    /// - returns: true if movement succeeded, otherwise returns false
+    public func move(_ filter: T, to destination: OrFilterGroup<T>) -> Bool {
+        return filterBuilder.move(filter: filter, from: group, to: AnyFilterGroup(destination))
+    }
+    
+    /// Replaces all the attribute by a provided one in group
+    /// - parameter attribute: attribute to replace
+    /// - parameter replacement: replacement attribute
+    public func replace(_ attribute: Attribute, by replacement: Attribute) {
+        return filterBuilder.replace(attribute, by: replacement, in: group)
+    }
+    
+    /// Replaces filter in group by specified filter replacement
+    /// - parameter filter: filter to replace
+    /// - parameter replacement: filter replacement
+    public func replace(_ filter: T, by replacement: T) {
+        return filterBuilder.replace(filter: filter, by: replacement, in: group)
+    }
+    
+    /// Removes all filters with specified attribute from group
+    /// - parameter attribute: specified attribute
+    public func removeAll(for attribute: Attribute) {
+        return filterBuilder.removeAll(for: attribute, from: group)
+    }
+    
+    @discardableResult public func remove(_ filter: T) -> Bool {
+        return filterBuilder.remove(filter, from: group)
+    }
+    
+    /// Removes a sequence of filters from group
+    /// - parameter filters: sequence of filters to remove
+    @discardableResult public func removeAll<S: Sequence>(_ filters: S) -> Bool where S.Element == T {
+        return filterBuilder.removeAll(filters, from: group)
+    }
+    
+    /// Removes all filters in group
+    public func removeAll() {
+        filterBuilder.removeAll(from: group)
+    }
+    
+    /// Removes filter from group if contained by it, otherwise adds filter to group
+    /// - parameter filter: filter to toggleE
+    public func toggle(_ filter: T) {
+        filterBuilder.toggle(filter, in: group)
+    }
+    
+    /// Constructs a string representation of filters in group
+    /// If group is empty returns nil
```

Comment:
Since this is the main function, I would consider adding one example of a filter string generated `(A:X OR B:Y) AND C:Z`